README.md

```diff
@@ -192,24 +199,24 @@ class BasicAuthBackend(backends.BasicAuthBackend):
 
 - `authenticated`
 
-### `BearerAuthBackend`
+### `BaseTokenAuth`
 
-Implementation of the [Bearer authentication scheme](https://tools.ietf.org/html/rfc6750), also known as _Token authentication_.
+Base implementation of token authentication, a simplified version of the [Bearer authentication scheme](https://tools.ietf.org/html/rfc6750).
 
 **Request header format**
 
 ```http
-Authorization: Bearer {token}
+Authorization: Token {token}
```

Comment:
Perhaps `Token` could be configurable in some way, eg by inheritance?

```python
class MySpecificTokenAuth(BaseTokenAuth):
    TOKEN_LABEL = "Bearer"
```

?

Reply:
Bearer authentication is actually a full-fledged standard — see [IETF RFC 6750](https://tools.ietf.org/html/rfc6750). Using `Bearer` should only be reserved to an authentication backend that *does* comply with that standard. As it is, `TokenAuth` does *not* comply, so I thought it would be better to use another scheme name altogether.